python/cuml/dask/preprocessing/label.py

```diff
@@ -228,8 +229,9 @@ def inverse_transform(self, y, threshold=None):
 
         f = [dask.array.from_delayed(
             inv_func(internal_model, part, threshold),
-            dtype=dtype, shape=(y.shape[0],), meta=meta)
+            dtype=dtype, shape=(cp.nan,), meta=meta)
```

Comment:
To be honest, I did not know about this nan trick and had to look it up - interesting!